packages/common/src/types/certificate.ts

```diff
@@ -158,8 +158,16 @@ export interface CheckboxCode {
   disabled?: boolean
 }
 
+export enum ConfigLayout {
+  ROWS = 'ROWS',
+  INLINE = 'INLINE',
+  COLUMN = 'COLUMN',
```

Comment:
Doesn't look like this is used, perhaps it can be removed?

Reply:
It is used in backend, so we might get error if not implemented 
I'm changing line 62 of radioGroup, which is what it's actually used for, so it becomes clearer (should be horizontal only has effect on ROWS-layout)
const shouldBeHorizontal = config.layout !== ConfigLayout.COLUMN && radiobuttons.length <= 2

Reply:
INLINE-layout is meant for very shoer labels, as in drivers licence categories: C, CE, D, DE etc. and then the spacing looks good. Maybe change the storybook to reflect this